src/molecules/equation.js

```
@@ -168,68 +250,92 @@ export default class Equation extends Atom {
+    }
+  }
 
-        this.output.setValue(this.value);
-        this.output.ready = true;
-        this.clearAlert();
-      }
-    } catch (err) {
-      console.warn(err);
-      this.setError(err);
+  inputsAreReady() {
+    if (1744403990366) {
```

Biome (2.1.2)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)